aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -962,7 +955,7 @@ void reconnectToSession() {
 
                 try {
                     final String mqttWebSocketURL = signer
-                            .getSignedUrl(endpoint, clientCredentialsProvider.getCredentials(),
```

Comment:
Can you add a comment on why this being changed from "endpoint" to "endpointWithHttpPort"?